src/main/java/io/cdap/plugin/format/MultiTableDBConfiguration.java

```diff
@@ -66,7 +66,7 @@ public void setTableAliases(List<String> tableAliases) {
   }
 
   public MultiTableConf getPluginConf() {
-    return GSON.fromJson(getConf().get(PLUGIN_CONF_FIELD), PLUGIN_CONF_TYPE);
+    return GSON.fromJson(getConf().getRaw(PLUGIN_CONF_FIELD), PLUGIN_CONF_TYPE);
```

Comment:
should also update all the other get() calls to be getRaw()

Reply:
Done.